.circleci/config.yml

```diff
@@ -47,6 +47,23 @@ jobs:
             - dist
             - .cache/Cypress
 
+  can_i_deploy:
+    environment:
+      PACT_BROKER_BASE_URL: "https://pact-broker-prod.apps.live-1.cloud-platform.service.justice.gov.uk"
+      PACT_BROKER_DISABLE_SSL_VERIFICATION: "true"
+    executor: hmpps/node
+    parameters:
+      environment:
+        type: string
+    steps:
+      - run:
+          name: Run Pact can-i-deploy
+          command: |
+            npx --package='@pact-foundation/pact-node' \
+                pact-broker can-i-deploy --pacticipant manage-recalls-ui --broker-base-url $PACT_BROKER_BASE_URL \
+                --version $CIRCLE_SHA1 \
+                --to deployed:<< parameters.environment >>
```

Comment:
Quick question more for my understanding - so how does this check for compatibility?  Is it looking for compatible versions with the tag `deployed:preprod` (for preoprod)?

Reply:
It uses the Pact matrix to check that the version of the pact file you are trying to deploy has been verified against the current version in whatever environment you specify (i.e the latest version that has been tagged in Pact with 'deployed:prod' for example)
 
So this version (currently in preprod) of the UI pact file is compatible with the following API versions:

https://pact-broker-prod.apps.live-1.cloud-platform.service.justice.gov.uk/matrix?q%5B%5Dpacticipant=manage-recalls-ui&q%5B%5Dversion=70eb961807b4596290066de95db25486c305b3df&q%5B%5Dpacticipant=manage-recalls-api&latestby=cvpv&limit=100

Reply:
Noice 👌